sources/postgres/data.go

```diff
@@ -186,13 +189,16 @@ func convInt64(val string) (int64, error) {
 // into a string representing a valid Spanner numeric.
 // Ideally we would just return a *big.Rat, but spanner.Mutation
 // doesn't currently support use of *big.Rat.
-// TODO: return *big.Rat when client library supports it.
-func convNumeric(val string) (string, error) {
-	r := new(big.Rat)
-	if _, ok := r.SetString(val); !ok {
-		return "", fmt.Errorf("can't convert %q to big.Rat", val)
+func convNumeric(conv *internal.Conv, val string) (interface{}, error) {
+	if conv.TargetDb == "experimental_postgres" {
```

Comment:
Use TARGET_EXPERIMENTAL_POSTGRES constant?

Reply:
We couldn't use the constant before because of circular dependency. Have now moved all constants to a top-level common package and now have updated to use constant as well.